packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -89,6 +90,10 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
     private pending: ISequencedDocumentMessage[] = [];
     private fetchReason: string | undefined;
 
+    // A boolean used to assert that ops are not being sent while processing another op.
+    private opsCurrentlyProcessing: boolean = false;
+    // Delta manager option enabling the above assertion
+    private readonly preventConcurrentOpSend: boolean = false;
```

Comment:
this probably needs to be read from a feature gate otherwise it's going to stay disabled until we write the code to enable it, right?

Follow how `maxOpSizeInBytesKey` is used inside `containerruntime.ts` as an example about feature gates.